### ISSUE TITLE
Login doesn't work on Firefox

### ISSUE THREAD

**Issue by gitxav** [the-road-to-graphql/fullstack-apollo-react-boilerplate#7]
Hi all.

A really nice app but unfortunately I get the following error with session on landing page:

![screenshot from 2018-12-31 17-33-21](https://user-images.githubusercontent.com/11687381/50569168-3757de80-0d24-11e9-9bac-5353f193c054.png)

My package.json in client folder:
{
  "name": "fullstack-apollo-react-boilerplate-project",
  "version": "0.1.0",
  "author": "Robin Wieruch <hello@rwieruch.com> (https://www.robinwieruch.de)",
  "dependencies": {
    "apollo-cache-inmemory": "^1.3.7",
    "apollo-client": "^2.4.4",
    "apollo-link": "^1.2.3",
    "apollo-link-error": "^1.1.1",
    "apollo-link-http": "^1.5.5",
    "apollo-link-ws": "^1.0.9",
    "apollo-utilities": "^1.0.24",
    "graphql": "^14.0.2",
    "graphql-tag": "^2.10.0",
    "history": "^4.7.2",
    "react": "^16.6.0",
    "react-apollo": "^2.2.4",
    "react-dom": "^16.6.0",
    "react-router-dom": "^4.3.1",
    "react-scripts": "2.1.0",
    "subscriptions-transport-ws": "^0.9.15"
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test --env=jsdom --passWithNoTests",
    "eject": "react-scripts eject"
  },
  "browserslist": [
    ">0.2%",
    "not dead",
    "not ie <= 11",
    "not op_mini all"
  ]
}

My package.json in server folder:
{
  "name": "fullstack-apollo-express-postgresql-boilerplate",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "engines": {
    "node": "10.11.0"
  },
  "scripts": {
    "start": "nodemon --exec babel-node src/index.js",
    "test:run-server": "TEST_DATABASE=mytestdatabase npm start",
    "test:execute-test": "mocha --require @babel/register 'src/**/*.spec.js'",
    "test": "echo \"No test specified\" && exit 0"
  },
  "keywords": [],
  "author": "Robin Wieruch <hello@rwieruch.com> (https://www.robinwieruch.de)",
  "license": "ISC",
  "devDependencies": {
    "@babel/core": "^7.1.6",
    "@babel/node": "^7.0.0",
    "@babel/preset-env": "^7.1.6",
    "@babel/register": "^7.0.0",
    "axios": "^0.18.0",
    "chai": "^4.2.0",
    "mocha": "^5.2.0",
    "nodemon": "^1.18.7",
    "morgan": "^1.9.1"
  },
  "dependencies": {
    "apollo-server": "^2.2.3",
    "apollo-server-express": "^2.2.3",
    "bcrypt": "^3.0.2",
    "cors": "^2.8.5",
    "dataloader": "^1.4.0",
    "dotenv": "^6.1.0",
    "express": "^4.16.4",
    "graphql": "^14.0.2",
    "graphql-iso-date": "^3.6.1",
    "graphql-resolvers": "^0.2.2",
    "jsonwebtoken": "^8.4.0",
    "pg": "^7.6.1",
    "sequelize": "^4.41.2",
    "uuid": "^3.3.2"
  }
}

I'm working with node 8.11.2 on ubuntu 16.04.

Thanks and I'll appreciate any hint.

**Comment by rwieruch** [comment]
Can you sign up with the server via GraphQL Playground when your server runs and you visit http://localhost:8000/graphpql ?

**Comment by gitxav** [comment]
Yes, I can signup from playground but when trying to login from landing page server responds with 400 error yet.
![screenshot from 2019-01-04 12-15-23](https://user-images.githubusercontent.com/11687381/50704428-f52ce680-101c-11e9-8418-27c417acf94e.png)

**Comment by rwieruch** [comment]
I tried it on my machine but cannot duplicate the issue :( Anyone else running into this?

**Comment by gitxav** [comment]
I tried project using Chrome and now it works nice :) . I used Firefox initially and I was unable to login and signup from landing page. I don't understand why.

**Comment by rwieruch** [comment]
That's a good hint. I will have to check it myself 👍 Thanks for trying it again!

**Comment by gitxav** [comment]
You are welcome :) . By the way the initial users data are not loaded in Firefox either:
'There are no messages yet ... Try to create one by yourself.'

**Comment by ACPK** [comment]
@rwieruch - Is this still an issue?

**Comment by rwieruch** [comment]
Not sure whether there are still issues on Firefox side.
